apps/api/src/persistence/database/arango-connection.provider.ts

```diff
@@ -149,9 +149,12 @@ export class ArangoConnectionProvider {
 
     async #createAllMissingCollections(): Promise<void> {
         await Promise.all(
-            getAllArangoCollectionIDs().map((collectionName) =>
-                this.#createCollectionIfNotExists(collectionName)
-            )
+            getAllArangoCollectionIDs()
+                /**
+                 * TODO Remove this and find a more graceful way to handle this.
```

Comment:
We need to make a story for injecting a second database specifically for queries. That's important.